Mutagen.Bethesda.Analyzers.Engine/Config/ConfigDirectoryProvider.cs

```diff
@@ -13,7 +13,18 @@ public IEnumerable<DirectoryPath> ConfigDirectories
         get
         {
             yield return currentDirectoryProvider.CurrentDirectory;
-            yield return dataDirectoryProvider.Path;
+
+            DirectoryPath? dataDirectory;
+            try
+            {
+                dataDirectory = dataDirectoryProvider.Path;
+            }
+            catch (Exception)
+            {
+                yield break;
```

Comment:
Let's swap this to `dataDirectory = null;`
and then follow up with a null check before the yield return.

This will be less prone to errors in future changes when, say, someone slaps in a 3rd source.  If they dont notice this `yield break`, the 3rd source might not be considered if the data dir is not present

Reply:
Good idea